website/README.md

```diff
@@ -1,44 +0,0 @@
-You will need Node 6 or newer in order to build the Jest website.
```

Comment:
horribly outdated, but something about `run yarn start` is probably useful?

Reply:
totally missed that I deleted this file 👍  thanks